File: javascript/part-2/array.js
var mobil = ["Avanza","Yaris","Alya"];
//Mengambil elemen pertama pada array
mobil[0] = 'bebek';
//Mengisi element array dengan push
mobil.push("Veloz");
//Membuat array kosong
var arrayKosong = [];
// menghapus paling belakang
mobil.pop()
// menghapus paling depan
mobil.shift()
//Mencetak semua elemen array
for(let i=0; i < mobil.length; i++){
console.log('Cetak mobil ke-' + i + ": " + mobil[i]);
}

let fruits = ["Apple", "Banana", "Cherry"]; 
console.log(fruits[0]); 
console.log(fruits[2]); 
fruits.push("mango");   
console.log(fruits); 
fruits[1] = "Blueberry";
console.log(fruits); 

File: python/array.py
mobil = ["Avanza", "Yaris", "Alya"]
# Mengambil elemen pertama pada list
mobil[0] = "bebek"
# Mengisi elemen list dengan "Veloz"
# mobil = ["Avanza", "Yaris", "Alya"]
mobil.append("Veloz")
# Membuat list kosong
array_kosong = []
# Mencetak semua elemen list
# mobil = ["Avanza", "Yaris", "Alya", "Veloz"]
#hapus
# del mobil[1]
#hapus paling belakang
# mobil.pop()
#hapus paling depan
mobil=mobil[1:]
for i in range(len(mobil)):
    print("Cetak mobil ke-" + str(i) + ": " + mobil[i])